subsys/bluetooth/controller/Kconfig

```diff
@@ -801,9 +801,14 @@ config BT_CTLR_SYNC_ISO
 
 config BT_CTLR_BROADCAST_ISO
 	bool
-	select BT_CRYPTO if BT_LL_SW_SPLIT
 	default BT_CTLR_ADV_ISO || BT_CTLR_SYNC_ISO
 
+config BT_CTLR_BROADCAST_ISO_ENC
+	bool
+	depends on BT_CTLR_CRYPTO_SUPPORT && BT_CTLR_BROADCAST_ISO
+	select BT_CRYPTO if BT_LL_SW_SPLIT
```

Comment:
It doesn't depend on `BT_CRYPTO`?

Should it have depended on `BT_CRYPTO` instead of `BT_CTLR_CRYPTO_SUPPORT` rather than `select`ing `BT_CRYPTO`?

Reply:
`BT_CRYPTO` is a module used by Host or Controller or both; and is a hidden Kconfig that needs to be selected by either Host or Controller.

Now, looking deep, maybe TinyCrypt is getting included, which is not needed by the Controller.... i will need to follow up in a separate PR.